src/components/Navigator/Navigator.tsx

```diff
@@ -40,83 +33,89 @@ const NavigatorTab: React.FC<NavigatorTabProps> = ({ label }) => {
   return <NavigatorTabWrapper>{label}</NavigatorTabWrapper>;
 };
 
-const NavigatorBodyInner: React.FC<NavigatorBodyInnerProps> = ({
-  items,
-  currentTime,
-}) => {
-  return items.map((cue) => {
-    return (
-      <NavigatorItem
-        text={cue.text}
-        startTime={cue.startTime}
-        t={currentTime}
-        key={cue.id}
-      />
-    );
-  });
+interface Cue {
+  id: number;
+  startTime: string;
+  endTime: string;
+  text: string;
+}
+
+interface NavigatorBodyInnerProps {
+  currentTime: number;
+  items: Cue[];
+}
+
+const NavigatorBodyInner: React.FC<NavigatorBodyInnerProps> = ({ items, currentTime }) => {
+  return (
+    <>
+      {items.map(({ id, text, startTime }) => (
+        <NavigatorItem label={text} startTime={startTime} t={currentTime} key={id} />
+      ))}
+    </>
+  );
```

Comment:
Wow, this makes so much more sense to me now. I'd like to clean up naming like we discussed but this great. 🧙